src/components/ClaimMasterPanel.js

```diff
@@ -72,6 +72,11 @@ class ClaimMasterPanel extends FormPanel {
       "claimForm.claimTypeReferSymbol",
       'R',
     );
+    this.numberOfAdditionalDiagnosis = props.modulesManager.getConf(
+      "fe-claim",
+      "claimForm.numberOfAdditionalDiagnosis",
```

Comment:
those two arguments could be also processed by constant. Please decide @wzglinieckisoldevelo but for me it could be good to use approach with const.